### DISCUSSION TITLE
[New Feature] Super-OmniCrusher

### DISCUSSION THREAD

**Discussion by TAK02** [Phobos-developers/Phobos#926] [Ideas and feedback]
### Description:

A Crush-class that can crush `OmniCrushResistant=yes` units.

IsTrain=yes would've been viable, but it has three issues:
* Scattering/Scatter Move when attacked/damaged doesn't happen.
* Can't be built from Factory without destroying that Factory
* Whenever a Train unit crushes a VehicleType with OmniCrushResistant it gets damaged.
### Possible Implementation:

```ini
[TechnoType]
UltiCrusher=no
UltiCrushResistant=no
```

**Comment by TAK02** [comment]
Note: intended use on my end is for a unit that is spawned via SW and vanishes after a while, so IsTrain=yes issues 2 and 3 aren't really problems for me.
Issue 1 is the problem for me. The thing this is intended for needs to roam the map via a weapon it fires on itself, and `IsTrain=yes` prevents any Scattering from happening.

**Comment by secsome** [comment]
I think it's better to give a crush level.
